src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -413,7 +413,7 @@
     <Field type="ulong" contentType="bytes" name="used" label="Used" description="Total memory occupied by objects within PLABs" />
     <Field type="ulong" contentType="bytes" name="undoWaste" label="Undo Wasted" description="Total memory wasted due to allocation undo within PLABs" />
     <Field type="ulong" contentType="bytes" name="regionEndWaste" label="Region End Wasted" description="Total memory wasted at the end of regions due to refill" />
-    <Field type="uint" contentType="bytes" name="regionsRefilled" label="Region Refills" description="Total memory wasted at the end of regions due to refill" />
+    <Field type="uint" name="regionsRefilled" label="Region Refills" description="Number of regions refilled" />
```

Comment:
Thanks for noticing. Maybe keep a `contentType="count"` though?

Reply:
I've changed the metadata analog to the numberOfRegions field of the G1HeapSummary event, which doesn't provide a content type. I can not find a count-like value which has a content type at all.

Reply:
Probably I misread something. Looks good.